src/TableFunctions/TableFunctionS3.cpp

```diff
@@ -29,9 +29,11 @@ void TableFunctionS3::parseArguments(const ASTPtr & ast_function, ContextPtr con
 
     const auto message = fmt::format(
         "The signature of table function {} could be the following:\n" \
+        " - url\n"
         " - url, format\n" \
         " - url, format, structure\n" \
         " - url, format, structure, compression_method\n" \
+        " - url, access_key_id, secret_access_key, format\n"
```

Comment:
And `url, access_key_id, secret_access_key`?

Reply:
Didn't support it, because we need to defferentiate it with `url, format, structure`, because both signatures have 3 arguments. But it's easy to do it, will implement it.